packages/web-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -13,17 +13,18 @@ import { useDropdownStyleProps } from './useDropdownStyleProps';
 
 const defaultProps = {
   enableAutoClose: true,
-  placement: Placements.BOTTOM_LEFT,
+  placement: Placements.BOTTOM_START,
 };
 
 export const Dropdown = (props: SpiritDropdownProps) => {
   const {
-    id = Math.random().toString(36).slice(2, 7),
     children,
-    renderTrigger,
     enableAutoClose,
     fullWidthMode,
+    id = Math.random().toString(36).slice(2, 7),
```

Comment:
I think the ID should better be injected or generated by a dedicated library like `nanoid`. Will think about it though.

Reply:
I know about this problem and I think the ID should be inserted by the developer. Maybe deprecate this and set it to be required.

Reply:
As agreed in person: since it's a new line of code, let's make the ID required instead. An issue will be filed to fix this across our code base.

Reply:
Update: it's not a new line of code, my bad 🙈. We will just drop a deprecation message here.

Reply:
https://jira.almacareer.tech/browse/DS-1170